sr_robot_launch/CMakeLists.txt

```diff
@@ -172,53 +172,53 @@ if (CATKIN_ENABLE_TESTING)
     find_package(rostest REQUIRED)
 
 #HAND TESTS
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_e hand_id:=rh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_e hand_id:=lh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_e_plus hand_id:=rh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_e_plus hand_id:=lh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_lite hand_id:=rh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_lite hand_id:=lh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_extra_lite hand_id:=rh)
-    add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_extra_lite hand_id:=lh)
+    #add_rostest(test/test_hand_joint_movement.test ARGS hand_type:=hand_e hand_id:=rh)
```

Comment:
Why commented but not deleted? In description you say:
> The gazebo tests will be done with nigh builds
Does that mean that these commented lines are defined somewhere else for night builds?

Reply:
I left them with comments as we want to check the available options for when we run the tests. Our automated platform will probably parse this and run them